doc/docs/openapi.md

```diff
@@ -213,6 +213,8 @@ Options can be customised by providing an instance of `OpenAPIDocsOptions` to th
 * `schemaName`: specifies how schema names are created from the full type name. By default, this takes the last
   component of a dot-separated type name. Suffixes might be added at a later stage to disambiguate between different
   schemas with same names.
+* `failOnDuplicateOperationId`: if set to `true`, the interpreter will throw an exception if it encounters two endpoints
+  with the same operation id. An OpenAPI document with duplicate operation ids is not valid. Code generators can silently drop duplicates.
```

Comment:
Maybe mention & link the endpoint verifier?

Reply:
Though I'm not 100% sure that such verification should be part of the interpreter. There are other errors that are checked by the `EndpointVerifier`, which would also cause invalid specs to be generated (like shadowing, duplicate methods, invalid body). So shouldn't we rather rely on the verifier to contain all the verifications?

Reply:
The feature (interpreter option) is opt-in, so not enabled by default. This is backwards compatible. 
The EndpointVerifier is not per-se focused on OpenAPI compatibility. I think it is fair to say the OpenAPI converter should have a role in this and preferably provide a safe conversion to an Either/Option/Try.

Reply:
Ok, let's leave it then

Reply:
I'll add some more docs